h2o/h2o.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+
+echo "BEGIN Stage 1 : Install H2O libraries and dependencies"
+apt-get update
+
+
+
+## Set Spark and Sparkling water versions
+H2O_SPARKLING_WATER_VERSION="3.28.0.3-1"
+SPARK_VERSION=$(spark-submit --version 2>&1 | sed -n 's/.*version[[:blank:]]\+\([0-9]\+\.[0-9]\).*/\1/p' | head -n1)
+echo $SPARK_VERSION
+
+
+
+## Download and unzip Sparking water Scala libraries
+SPARKLING_WATER="http://h2o-release.s3.amazonaws.com/sparkling-water/spark-$SPARK_VERSION/$H2O_SPARKLING_WATER_VERSION-$SPARK_VERSION/sparkling-water-$H2O_SPARKLING_WATER_VERSION-$SPARK_VERSION.zip"
+echo $SPARKLING_WATER
+PYSPARKLING_WATER="h2o_pysparkling_$SPARK_VERSION"
+echo $PYSPARKLING_WATER
+
+cd /usr/lib/
+wget $SPARKLING_WATER
+unzip "sparkling-water-$H2O_SPARKLING_WATER_VERSION-$SPARK_VERSION.zip"
+
+H2O_SPARKLING_WATER_HOME="/usr/lib/sparkling-water-$H2O_SPARKLING_WATER_VERSION-$SPARK_VERSION"
+echo $H2O_SPARKLING_WATER_HOME
+
+
+
+## Update $TOPDIR variable in sparkling scripts and add home variable
+sed -i "2iH2O_SPARKLING_WATER_HOME=\"$H2O_SPARKLING_WATER_HOME\"" $H2O_SPARKLING_WATER_HOME/bin/sparkling-shell
+sed -i 's/TOPDIR=$(cd "$(dirname "$0")\/.."; pwd)/TOPDIR=$H2O_SPARKLING_WATER_HOME/g' $H2O_SPARKLING_WATER_HOME/bin/sparkling-shell
+
+sed -i "2iH2O_SPARKLING_WATER_HOME=\"$H2O_SPARKLING_WATER_HOME\"" $H2O_SPARKLING_WATER_HOME/bin/pysparkling
+sed -i 's/TOPDIR=$(cd "$(dirname "$0")\/.."; pwd)/TOPDIR=$H2O_SPARKLING_WATER_HOME/g' $H2O_SPARKLING_WATER_HOME/bin/pysparkling
+
+## create soft link entries for default
+ln -s $H2O_SPARKLING_WATER_HOME/bin/sparkling-shell /opt/conda/default/bin/sparkling-shell
+ln -s $H2O_SPARKLING_WATER_HOME/bin/pysparkling /opt/conda/default/bin/pysparkling
+
+
+## Install python packages
+PY_PATH=`ls -al /opt/conda/default`
+echo $PY_PATH
+
+if [[ $PY_PATH == *"anaconda"* ]]; then
+        echo "anaconda"
+        /opt/conda/default/bin/pip install -U requests tabulate future colorama scikit-learn google-cloud-bigquery google-cloud-storage
+        /opt/conda/default/bin/pip install -U h2o $PYSPARKLING_WATER
+        # conda install failing : "Solving environment: ...working... failed"
+        # /opt/conda/default/bin/conda install tabulate future colorama scikit-learn
+        # /opt/conda/default/bin/conda install --override-channels -c main -c conda-forge google-cloud-bigquery google-cloud-storage
+        # /opt/conda/default/bin/conda install -c h2o $PYSPARKLING_WATER
```

Comment:
Remove commented code.

Reply:
Removed commented code. But have kept the python check incase we want to support `conda install` in future.